cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,84 @@
+import { generateUser } from '../support/generate';
 /// <reference types='cypress' />
 
+const user = generateUser();
+
+const rowsChoose = user.rows[Math.floor(Math.random() * user.rows.length)];
+
+const worker = 'Cierra';
+
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('pagination testing', () => {
+    cy.get('.-pageInfo').should('contain', 'Page');
+
+    cy.get('select').select(rowsChoose);
+
+    for (let i = 0; i < parseInt(rowsChoose.split(' ')[0]); i++) {
+      cy.get(`:nth-child(${i + 1})`).should('exist');
+    }
+  });
+
+  it('should able to add and delete new worker', () => {
```

Comment:
separate this test on 2 different - adding worker and deleting worker

Reply:
Splited